rest_framework/views.py

```diff
@@ -118,8 +119,19 @@ def as_view(cls, **initkwargs):
         This allows us to discover information about the view when we do URL
         reverse lookups.  Used for breadcrumb generation.
         """
+        if isinstance(getattr(cls, 'queryset', None), models.QuerySet):
```

Comment:
This should probably be `models.query.QuerySet`. Seems like `QuerySet` wasn't expose as `models.QuerySet` until 1.7.